svf-llvm/lib/SVFIRBuilder.cpp

```diff
@@ -282,11 +282,15 @@ bool SVFIRBuilder::computeGepOffset(const User *V, AccessPath& ap)
 
     bool isConst = true;
 
+    bool addedPointerType = false;
     for (bridge_gep_iterator gi = bridge_gep_begin(*V), ge = bridge_gep_end(*V);
             gi != ge; ++gi)
     {
         const Type* gepTy = *gi;
         const SVFType* svfGepTy = LLVMModuleSet::getLLVMModuleSet()->getSVFType(gepTy);
+
+        assert((!addedPointerType || !svfGepTy->isPointerTy()) && "multiple pointer type?");
```

Comment:
(addedPointerType && svfGepTy->isPointerTy()) == false

addedPointerType => prevPtrOperand

"multiple pointer type?" => "Expect no more than gep operand to be of a pointer type"